### PR TITLE
[CoC] Adds a code of conduct, thanks for the inspiration @indirect

### PR DESCRIPTION
**Pull request by orta**
Thanks bundler team also

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +41,7 @@ we welcome new contributors with joy.
 [Guides](http://guides.cocoapods.org) | Everything you want to know about CocoaPods.
 [Changelog](https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md) | See the changes introduced in each CocoaPods version.
 [New Pods RSS](http://feeds.cocoapods.org/new-pods.rss) | Don't miss any new Pods.
+[Code of Conduct](CODE_OF_CONDUCT.md) | Find out the standards we hold ourselves to
```
Comment (segiddins) [inline README.md]: Trailing period.

**Comment by segiddins** [comment]
Is this lifted verbatim from bundler? If so, :+1:

**Comment by orta** [comment]
Yes, it's CC0, so we can just take it, then I added them to the credits.

**Comment by alloy** [comment]
Very :+1:

**Comment by orta** [comment]
ok not mergable yet -> `spec/cocoapods-integration-specs` changes

**Comment by orta** [comment]
ok, fixed

**Comment by orta** [comment]
got all the :+1:s I need - merging.

**Pull request merged**